ben02/thermf_ben02.F

```diff
@@ -371,18 +385,16 @@ subroutine thermf_ben02(m,n,mm,nn,k1m,k1n)
 c --- --- Accumulate the runoff in a reservoar to delay the discharge
 c --- --- into the ocean (by nrfets days approximately 1/e of runoff
 c --- --- added will by discharged).
-          rnfres(i,j)=rnfres(i,j)+rnf(i,j)
-          rnfflx(i,j)=rnfres(i,j)*rnf_fac
+          rnfres(i,j)=rnfres(i,j)+rnfins(i,j)
```

Comment:
Change runoff routine?

Reply:
It is just a change of variable name to make the forcing fields defined in ben02* experiments configurations and when coupled to NorESM more consistent. This does not change results or functionality.